regulation-worker/internal/delete/api/api.go

```diff
@@ -76,6 +108,22 @@ func (api *APIManager) Delete(ctx context.Context, job model.Job, destConfig map
 		return model.JobStatusFailed
 	}
 
+	// Refresh Flow Start
+	if delAccountIdExists {
+		respParams := &handleRefreshFlowParams{
+			jobResponses: jobResp,
+			secret:       accountSecretInfo.Account.Secret,
+			workspaceId:  job.WorkspaceID,
+			accountId:    rudderUserDeleteAccountId.(string),
+			destName:     destDetail.Name,
+		}
+		refreshFlowStatus := api.handleRefreshFlow(respParams)
```

Comment:
I think this flow of handling refresh could be improved if we know the status code of the response returned.

Reply:
The refactoring for refresh flow handling is complete. Can you please check and verify ?